src/stores/store/eth1/local/local.go

```diff
@@ -406,6 +418,16 @@ func getEIP712EncodedData(typedData *core.TypedData) (string, error) {
 	return fmt.Sprintf("\x19\x01%s%s", domainSeparatorHash, typedDataHash), nil
 }
 
+// getEIP191EncodedData encodes the given message that can be later recovered
+// with the given validator. following EIP-191 spec version 0x0
+func getEIP191EncodedData(address string, msg []byte) (eipFormat, msgHash []byte, err error) {
```

Comment:
Having `msgHash` as part of the response of this method seems to not have any value added, do you think you could remove it?

Reply:
This is the first place where we are able to compute it, I thought it would be good to do so here.